aws-android-sdk-cognitoauth/src/main/java/com/amazonaws/mobileconnectors/cognitoauth/AuthClient.java

```diff
@@ -153,8 +161,10 @@ protected void setUsername(final String username) {
      *     state in the redirect uri to fetch the stored proof-key.
      * </p>
      * @param showSignInIfExpired true if the web UI should launch when the session is expired
+     * @param activity The activity to launch the sign in experience from.
+     *                 This must not be null when showSignInIfExpired is true.
```

Comment:
What happens if `activity` is null and `showSignInIfExpired` is true? Is this relationship enforced anywhere?

Reply:
Question resolved in-person